src/benchcab/fluxsite.py

```diff
@@ -215,6 +225,7 @@ def run(self):
         try:
             self.run_cable()
             self.add_provenance_info()
+            self.state.set("done")
```

Comment:
Ideally, we would need to add a test on `run()` to test the state is set correctly. If anything that would document where the state is set. It would likely be as simple as ` test_task_is_done_on_success` in test_comparison.py.

Reply:
I would like to avoid writing a test for the `run()` function if possible as there would almost be no functionality being tested other than state being set to `done`. It would be an integration test where we would have to mock everything the function does. It's probably a symptom of my poorly designed code 😆

Reply:
I'm not sure I completely agree, especially since ComparisonTask.execute_comparison() got a test on setting the state, but that's not worth a lengthy discussion.